Added community colleges and minor updates

dmoruzzi opened the pull request:
Hello. I attempted to follow the contributions guidelines as close as possible. Apologies if I did not do something correctly. Also sorry for not following the provided template. Due to the number of colleges added, it is not practical in this situation. 

All the community colleges in Alaska, Michigan, North Dakota, South Dakota, and Vermont were added, according to the list provided by [Community College Review](http://communitycollegereview.com/). The domain of ITT Technical Institute was corrected to include a hyphen. Removed the domain of "edu.tf" as it is nonexistent. Removed the domain "edu.vlerick.com" as "vlerick.com" is present. Lastly, corrected the domain of the International Turkmen-Turkish University.

ReidWeb commented:
Hi David,

Thanks for opening this.

Fine not to use the template in this case. However it would be much appreciated if you could do out a checklist like this:

- [ ] [Oxford University](http://www.ox.ac.uk/)
- [ ] [IKing's College London](http://www.kcl.ac.uk/index.aspx)
- [ ] [University of Cambridge](https://www.cam.ac.uk/)
...

this will make it much easier to review the institutions as I can run down the list and check them off as I go

dmoruzzi commented:
OK. It'll take me a couple of minutes.

dmoruzzi committed:
Community colleges and minor updates

All the community colleges in Alaska, Michigan, North Dakota, South
Dakota, and Vermont were added, according to the list provided by
Community College Review. The domain of ITT Technical Institute was
corrected to include a hyphen. Removed the domain of "edu.tf" as it is
nonexistent. Removed the domain "edu.vlerick.com" as "vlerick.com" is
present. Lastly, corrected the domain of the International
Turkmen-Turkish University.
